src/components/CActorArgument.vue

```diff
@@ -0,0 +1,296 @@
+            :only-date="valueType === 'datetime.date'"
+          />
+        </div>
+        <input
+          v-else
+          v-model="typedValue"
+          :type="
+            valueType === 'int' || valueType === 'float'
+              ? 'number'
+              : valueType === 'bool'
+              ? 'checkbox'
+              : 'text'
+          "
+          :step="type === 'float' ? 'any' : '1'"
+          :class="
+            valueType === 'bool'
+              ? 'ml-0 w-10 h-10 w-1/2'
+              : 'appearance-none w-1/2 block h-full mb-4 bg-gray-200 text-gray-700 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-500'
+          "
+          :required="valueType === 'int' || valueType === 'float'"
+        />
+        <button
+          type="submit"
+          class="bg-blue-500 text-white flex-shrink-0 font-bold py-2 px-4 rounded h-full w-32"
+          :disabled="isInvalid"
+          :class="{
+            'hover:bg-blue-700': !isInvalid,
+            'opacity-50': isInvalid,
+            'cursor-not-allowed': isInvalid
+          }"
+        >
+          + Add
+        </button>
+      </form>
+      <div v-for="(el, index) in value" :key="index" class="h-10 w-full flex">
+        <label
+          class="appearance-none block w-full h-full mb-4 bg-gray-200 text-gray-700 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
+        >
+          {{ index }} : {{ el }}
+        </label>
+        <button
+          type="button"
+          @click="$delete(value, index, 1)"
+          class="bg-red-500 text-white font-bold py-2 px-4 rounded h-full flex-shrink-0 w-32"
+        >
+          Remove
+        </button>
+      </div>
+    </div>
+    <VueCtkDateTimePicker
+      v-else-if="type === 'datetime'"
+      v-model="value"
+      format="YYYY-MM-DD HH:mm"
+    />
+    <VueCtkDateTimePicker
+      v-else-if="type === 'date'"
+      v-model="value"
+      formatted="ll"
+      only-date
+      format="YYYY-MM-DD"
+    />
+    <textarea
+      v-else
+      class="block appearance-none w-full mb-4 bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
+      v-model="value"
+    />
+  </div>
+</template>
+
+<script>
+import draggable from 'vuedraggable';
+import utils from '@/utils';
+
+export default {
+  name: 'CActorArgument',
+  props: {
+    name: String,
+    type: String
+  },
+  components: {
+    draggable
+  },
+  data: function() {
+    let value;
+    let listType = '';
+    let typedValue = '';
+    let keyValue = '';
+    switch (this.type) {
+      case 'int':
+      case 'float':
+        value = 0;
+        break;
+      case 'bool':
+        value = false;
+        break;
+      case 'dict':
+        value = {};
+        break;
+      case 'list':
```

Comment:
`type.match(/^List/)` as well I guess. Same for `dict`

Reply:
It's not possible here as these cases are not meant for typing.List and typing.Dict but for list and dict with unspecified inner types.